deepmd/infer/deep_property.py

```diff
@@ -141,5 +141,13 @@ def get_intensive(self) -> bool:
         """Get whether the property is intensive."""
         return self.deep_eval.get_intensive()
 
+    def get_property_name(self) -> Union[list[str], str]:
+        """Get the names of the properties."""
+        return self.deep_eval.get_property_name()
```

Comment:
There is no `get_property_name` or `get_property_dim` defined in `DeepEvalBackend`.

Reply:
Done